src/components/slider/slider.ts

```diff
@@ -215,6 +228,80 @@ export class MdSlider implements AfterContentInit {
   }
 
   /**
+   * Calls the appropriate method for calculating the tick separation depending on what the tick
+   * interval is. If there is no tick interval or the interval is set to something other than a
+   * number or 'auto', nothing happens.
+   */
```

Comment:
This function description should just be something like "Calculates the separation in pixels of tick marks...". The fact that it delegates to another function is not something you need to mention in the description.